src/modules/integrator-guide/pages/compose-configuration/page-blocks.adoc

```diff
@@ -743,3 +743,48 @@ Defines the report the page bock will use when showing data.
 Defines the display element from the selected report the page block will use when showing data.
 
 |===
+
+[#page-block-navigation]
+== Navigation
+
+The *navigation page block* allows you to show navigation, which can easily navigate through different pages.
```

Comment:
This makes me question the feature on what exactly it does.
Perhaps something along the lines of what I've done in the changelog file would work better? 

`Navigation page block allows you to define custom navigation inside your Low Code applications. The navigation can be an addition to the pre-defined one or a complete replacement.`